packages/forms-web-app/tests/unit/controllers/appellant-submission/site-location.test.js

```diff
@@ -90,7 +90,7 @@ describe('controllers/appellant-submission/site-location', () => {
       });
     });
 
-    it('should redirect to `/appellant-submission/site-ownership` if valid', async () => {
+    it('should redirect to `/appeal-householder-decision/site-ownership` if valid', async () => {
```

Comment:
please can the dir here be updated to reflect `appeal-householder-decision` - thanks